pkg/storaged/stratis/pool.jsx

```diff
@@ -106,7 +109,7 @@ function create_fs(pool) {
                        }),
             SelectOne("at_boot", _("At boot"),
                       {
-                          value: "nofail",
+                          value: client.in_anaconda_mode() ? "local" : "nofail",
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19352-20231205-120952-f0d04c46-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)